editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/internal/l10n/messages.properties

```diff
@@ -70,6 +70,8 @@ autoLayoutActionDescription=Automatic layout of the diagram
 autoLayoutActionTooltip=Does an automatic layout of the diagram
 beanClassExistsValidatorErrorBeanClassMandatory=Bean class name is mandatory.
 beanClassExistsValidatorErrorBeanClassMustExist=Bean class must exist in the Fuse project.
+beanConfigUtilNoMethodsAvailable=No methods available.
+beanConfigUtilSelectStaticPublicMethod=Select a static, public method from the list.
```

Comment:
I'm not english native speaker so i might be wrong, but I think that I would better understand with one the following:
- Select a static and public method from the list.
- Select a public static method from the list.